charts/nextcloud/ci/ct-all-enabled-values.yaml

```diff
@@ -0,0 +1,11 @@
+nextcloud:
+  configs:
+    my.config.php: |-
+      <?php
+
+nginx:
+  enabled: true
+  config:
+    custom: |-
```

Comment:
it is just my test-value here

Reply:
maybe that is better

Reply:
`ci/*-values.yaml` is used by chart-testing. it try to lint / install with every of that files.

your chart-testing github action pipeline try an **install** in a kubernetes clusters.
So it checks, if all helm generated resources also comes up (like helm --wait).
But with my current example with an customize nginx config (``).
is put correct in the configmap and to the pod, but the nginx does not reach the state running (because the nginx config is invalid with my test values).

I just use the ct **lint** command on developing, so i have not seen, that this chart works (and the part is correct put together)

Reply:
i remove the ct-values for #523 - to make this PR smaller